### PR TITLE
[Apicurio] Generated JAX-RS for Registry API

### PR DESCRIPTION
**Pull request by EricWittmann**
The Apicurio tool was used to generate an API project from an OpenAPI definition.  Please review this pull request, modify as necessary, and merge!

### CODE REVIEW

File: app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java [hunk 1]
```diff
@@ -0,0 +1,294 @@
+
+package io.apicurio.registry.rest.beans;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import com.fasterxml.jackson.annotation.JsonCreator;
```
Comment (alesj) [inline app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java]: Atm I've used jaxb, as Quarkus examples point in that direction.
But we can change it to Jackson ... if we know how. :-)

Comment (EricWittmann) [reply app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java]: I'll work on this as I evolve the jax-rs layer.  I've requested Jackson support for this release from the quarkus team and they've agreed.  I'm using a 3rd party library in Apicurio Studio to generate the bean classes from the JSON Schema data types defined in the OpenAPI spec.   We could switch to jaxb but I'd need to either user a diff. library or try to change the one I'm using to switch from jackson to jaxb...

Comment (EricWittmann) [reply app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java]: That said, our Data Model library in Apicurio (which we'll be using to process the OpenAPI and AsyncAPI artifacts) requires Jackson.  So we require Jackson in Quarkus for the registry anyway.

Comment (alesj) [reply app/src/main/java/src/main/java/io/apicurio/registry/rest/beans/ArtifactMetaData.java]: Why requiring Jackson? Or why JAXB is not good enough?
Or Quarkus really doesn't support Jackson atm?

**Comment by EricWittmann** [comment]
Closing this PR - I will generate a new one, test it, and then push.

**Pull request closed without merging**